kolibri/core/assets/src/heartbeat.js

```diff
@@ -23,12 +23,10 @@ const timeoutReconnectTime = 60;
 
 const minReconnectTime = 5;
 
+const activeDelay = 240000;
```

Comment:
few minor thoughts. Maybe...

* constant variable names to `SCREAMING_SNAKE_CASE`
* indicate here how this value is chosen? E.g. "the server closes the session after 10 minutes so we should poll at least every 5 minutes"
* store in seconds for consistency with reconnect time

Reply:
Have addressed this in latest commits.